src/Command/CompileCommand.php

```diff
@@ -0,0 +1,148 @@
+<?php
+declare(strict_types=1);
+
+namespace Cake\TwigView\Command;
+
+use Cake\Console\Arguments;
+use Cake\Console\BaseCommand;
+use Cake\Console\ConsoleIo;
+use Cake\Console\ConsoleOptionParser;
+use Cake\TwigView\Filesystem\Scanner;
+use Cake\TwigView\View\TwigView;
+use Exception;
+
+class CompileCommand extends BaseCommand
+{
+    /**
+     * @var \Cake\TwigView\View\TwigView
+     */
+    private $twigView;
+
+    /**
+     * @inheritdoc
+     */
+    protected function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
+    {
+        $parser->addArgument('type', [
+            'required' => true,
+            'choices' => ['all', 'file', 'plugin'],
+            'help' => 'The type you want to compile.',
+        ]);
+
+        $parser->addArgument('target', [
+            'required' => false,
+            'help' => 'The file or plugin you want to compile.',
+        ]);
+
+        return $parser;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+        $type = $args->getArgumentAt(0);
+
+        if (!in_array($type, ['all', 'file', 'plugin', true])) {
```

Comment:
Unfortunately, this code is unreachable/untestable because Console validates the choices before executing.